apps/dashboard/src/main/java/com/akto/action/DashboardAction.java

```diff
@@ -255,6 +249,48 @@ public String markConnectionAsSkipped(){
         }
     }
 
+    private String email;
+    private String username;
+    private String organization;
+    private static final String USERNAME_REGEX = "^[\\w\\s-]{1,}$";
```

Comment:
use precompiled pattern and not matches. matches compiles regex everytime the function is called, might affect efficiency